docs/getting_started/creating_an_admin.rst

```diff
@@ -5,6 +5,10 @@ You've been able to get the admin interface working in :doc:`the previous
 chapter <installation>`. In this tutorial, you'll learn how to tell SonataAdmin
 how an admin can manage your models.
 
+.. note::
+    This article assumes you are using Symfony 4. Using Symfony 2.8 or 3 will require
```

Comment:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Reply:
Done